resources/definitions/zonestar_z5s_printer.def.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "ZONESTAR Z5S",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "ZONESTAR",
+        "manufacturer": "ZONESTAR",
+        "file_formats": "text/x-gcode",
+        "platform_offset": [ 0, 0, 0],
+        "machine_extruder_trains":
+        {
+            "0": "zonestar_printer_extruder_0"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "ZONESTAR Z5S" },
+        "layer_height": { "default_value": 0.2 },
+        "machine_extruder_count": { "default_value": 1 },
+        "machine_heated_bed": { "default_value": true },
+        "machine_center_is_zero": { "default_value": false },
+        "machine_height": { "default_value": 250 },
```

Comment:
According to [this sales page](https://www.gearbest.com/3d-printers--3d-printer-kits/pp_009772894486.html), the height is 240mm, not 250mm. Which of you is correct?